azure-core/azure/core/pipeline/transport/aiohttp.py

```diff
@@ -49,21 +50,30 @@ class AioHttpTransport(AsyncHttpTransport):
     """AioHttp HTTP sender implementation.
     """
 
-    def __init__(self, configuration=None, *, loop=None):
+    def __init__(self, configuration=None, *, session=None, loop=None, session_owner=True):
         self._loop = loop
-        self.session = None
-        self.config = configuration
+        self._session_owner = session_owner
```

Comment:
Offhand I'm not sure I'd know without looking at the implementation whether `session_owner=True` signifies that  the caller is the owner, or the transport object is the owner. Maybe `cleanup_session` or `take_ownership`?

Reply:
I guess `session_owner` also implies creating a session, if needed, so maybe those suggestions aren't great. I'd just make it clear in in the docstring what is implied by this setting